melange-opam-template.opam

```diff
@@ -13,8 +13,8 @@ depends: [
   "melange" {>= "1.0.0"}
   "reactjs-jsx-ppx" {>= "1.0.0"}
   "reason-react" {>= "0.11.0"}
-  "ocaml-lsp-server" {dev}
-  "dot-merlin-reader" {dev}
+  "ocaml-lsp-server" {with-dev-setup}
+  "dot-merlin-reader" {with-dev-setup}
```

Comment:
I think this line can be removed altogether, this was required by melange before the dune integration was implemented.

Reply:
The `dot-merlin-reader` line?

Reply:
Yes.

Reply:
(Can be done in another PR if you prefer)